pkg/schemas/security_integration_property_gen.go

```diff
@@ -0,0 +1,41 @@
+// Code generated by sdk-to-schema generator; DO NOT EDIT.
+
+package schemas
+
+import (
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+// ShowSecurityIntegrationPropertySchema represents output of SHOW query for the single SecurityIntegrationProperty.
+var ShowSecurityIntegrationPropertySchema = map[string]*schema.Schema{
+	"name": {
+		Type:     schema.TypeString,
+		Computed: true,
+	},
+	"type": {
+		Type:     schema.TypeString,
+		Computed: true,
+	},
+	"value": {
+		Type:     schema.TypeString,
+		Computed: true,
+	},
+	"default": {
+		Type:     schema.TypeString,
+		Computed: true,
+	},
+}
+
+var _ = ShowSecurityIntegrationPropertySchema
+
+func SecurityIntegrationPropertyToSchema(securityIntegrationProperty *sdk.SecurityIntegrationProperty) map[string]any {
```

Comment:
Why passing a pointer?

Reply:
This is generated and it's everywhere a pointer, which is basically a bit faster. So why not a pointer?